package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "input-plugin-textarea-limited",
-  "version": "0.0.0",
+  "keyword": "punchcard-cms",
```

Comment:
keyword already exists (line 42+ below) and all input plugins must have both `punchcard-cms` and `input-plugin` as keywords.